src/js/shield_defs.js

```diff
@@ -1080,10 +1072,10 @@ export function loadShields() {
     textLayout: textConstraint("southHalfEllipse"),
     textColor: Color.shields.white,
     padding: {
-      left: 4,
-      right: 4,
+      left: 5,
+      right: 5,
       top: 6,
```

Comment:
Reducing the top padding by 1px seems to give a better result:
```suggestion
      top: 5,
```
![image](https://user-images.githubusercontent.com/3254090/227676172-6575aa10-af4f-4816-8d55-006c2a69bd28.png)